doc/man3/flux_shell_service_register.adoc

```diff
@@ -0,0 +1,39 @@
+flux_shell_service_register(3)
+==============================
+:doctype: manpage
+
+NAME
+----
+flux_shell_service_register - Register a service handler for `method` in the shell.
+
+SYNOPSIS
+--------
+
+ #include <flux/shell.h>
+
+ int flux_shell_service_register (flux_shell_t *shell, 
+                                  const char *method, 
+                                  flux_msg_handler_f cb, 
+                                  void *arg);
+
+DESCRIPTION
+-----------
+
+Wrapper for `shell_svc_register()`.  Returns -1 on failure and sets 
+errno to EINVAL if `shell`, `method` or `cb` is NULL.  Otherwise 
+returns the results of the underlying call to `shell_svc_register()`.
```

Comment:
Split out `RETURN VALUE` and `ERRORS` sections.

Instead of referencing a shell internal function, you can just repeat the wording in the synopsis.
Or perhaps

Register a message service handler `cb` to be invoked for the shell method `method`. Messages are sent to this service using the topic string `shell-JOBID.method`, where `JOBID` is the jobid under which the current shell is executing.

Reply:
Already done.